src/app/item-popup/ItemStat.m.scss

```diff
@@ -79,27 +79,29 @@
 
 // Colors for the stat bars
 .masterworkStatBar {
-  background-color: $orange;
+  background-color: var(--theme-accent-primary);
```

Comment:
Since these stat bar colours relate to specific contributors, I don't think they should change with the theme. Could we use a gold colour for Masterwork-affected stats and a blue colour for mod-affected stats (which lines up with the new `Font of ...` mods)?